src/components/modules/dashboard/offer/ui/dialogs/LoanRequestDetailDialog.tsx

```
@@ -25,117 +35,340 @@ const LoanRequestDetailDialog = ({
+            value="milestones"
+            className="p-6 pt-4 focus-visible:outline-none focus-visible:ring-0"
+          >
+            <div className="space-y-4">
+              <div className="flex items-center gap-2">
+                <Milestone className="h-5 w-5 text-amber-600" />
+                <h3 className="font-medium">Loan Milestones</h3>
+              </div>
+
+              {selectedRequest.milestones?.length > 0 ? (
+                <div className="space-y-3">
+                  {selectedRequest.milestones?.map((m: any, idx: number) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)